app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -22,55 +22,12 @@ export const CONTACT_FORM_VALIDATION_SCHEMA = yup.object().shape({
     .string()
     .email("Invalid email format")
     .required("Email is required"),
-  role: yup.object().required("Role is required").nullable(),
+  role: yup
+    .object()
+    .nullable()
+    .shape({
+      label: yup.string().oneOf(ROLES),
+      value: yup.string().oneOf(ROLES),
```

Comment:
The value should be the sluggified version of the label. Because we will have use cases with the value like say string comparison. So it's preferred to keep the value as a slug. Label can be kept as it's.

Reply:
So i should convert label:"UX Designer" to value:"UX_Designer" ?

Reply:
Slugs are always hyphenated(`-`). On second thoughts, rather than using a slug, it's better to use a camelCase version for the `value`. This way we can utilise the value as a JavaScript variable if need be.

```js
{
 label: "Email Field",
 value: "emailField"
}
```

Reply:
I have made necessary changes and the changes will be reflected in next PR.

Reply:
Converted all occurences of such values to their respective label converted to camel case. The changes are reflected in constants.js file in both Contacts folder and Notes folder .